package.json

```diff
@@ -19,8 +20,8 @@
     "typescript:esbuild": "node scripts/unit-typescript-esbuild.js",
     "typescript:vitest": "vitest run",
     "typescript:vitest:dev": "vitest",
-    "unit": "node scripts/unit.js",
-    "unit:with-modules": "tap test/issues/*/test.js test/commonjs/*.js test/module/*.js test/typescript/*.ts"
+    "unit": "tap plugin rm @tapjs/typescript && node scripts/unit.js",
```

Comment:
The typescripts files and other scripts are launched first and tests finish with `node scripts/unit.js`.